.pre-commit-config.yaml

```diff
@@ -33,6 +33,7 @@ repos:
     rev: v2.2.6
     hooks:
       - id: codespell
+        args: ["--skip=*.min.css.map, *.min.css"]
```

Comment:
@huong-li-nguyen  here we might have to change accordingly

Reply:
Ah yes - if you've renamed the file, then we just have to reflect the name that we use for the map file.